.circleci/config.base.yml

```diff
@@ -103,8 +122,21 @@ jobs:
               - equal: [*windows-e2e-executor, << parameters.os >>]
           steps:
             - checkout
-            - run: nvm install 12.22.7
-            - run: nvm alias default 12.22.7
+            - run:
+                name: Install Node 18
+                command: |
+                  nvm version
+                  nvm install 18.15.0
+                  nvm use 18.15.0
```

Comment:
I added the `node/install` step for the android executor because it does not have `nvm` included. https://github.com/aws-amplify/amplify-codegen/blob/main/scripts/split-e2e-tests.ts#L135

I can't remember what happened when trying node/install on all the jobs, but it would be worth trying to possibly simplify the config.

Reply:
I've tried using the `node` orb for windows and it fails. Refer [this job](https://app.circleci.com/pipelines/github/aws-amplify/amplify-codegen/2904/workflows/b146be4a-750f-4272-a9cd-f972c9133496/jobs/16030) for error info.

Reply:
```
$ nvm alias default 18.15.0
$ nvm use
```
is identical to `nvm use 18.15.0`. The later is concise.